Framework/Core/test/test_DataSampling.cxx

```diff
@@ -43,8 +43,8 @@ BOOST_AUTO_TEST_CASE(DataSamplingSimpleFlow)
 
   std::string configFilePath = std::string(getenv("O2_ROOT")) + "/share/tests/test_DataSampling.json";
   std::cout << "config file : "
-            << "json://" << configFilePath << std::endl;
-  DataSampling::GenerateInfrastructure(workflow, "json://" + configFilePath);
+            << "json:/" << configFilePath << std::endl;
+  DataSampling::GenerateInfrastructure(workflow, "json:/" + configFilePath);
```

Comment:
why is this change needed ?

Reply:
I noticed that Configuration, when it couldn't find a file, was printing the path with double `/` at the beginning, like so:
```
Unable to read JSON file: //home/pkonopka/alice/sw/slc7_x86-64/QualityControl/use-workflow-merging-1/etc/basicaaa.json
```
So as I understand, the preferred or default way is to use one `/` in `json:/`. That being said, both alternatives work when there is a file available at specified path.

Reply:
ok